server/db/models/index.js

```diff
@@ -0,0 +1,21 @@
+const Order = require('./Order');
+const Product = require('./Product');
+const Session = require('./Session');
+const User = require('./User');
+const OrderProduct = require('./OrderProduct');
+
+User.hasMany(Order);
+Order.belongsTo(User);
+
+Order.hasMany(OrderProduct);
+OrderProduct.belongsTo(Order);
+
+Product.hasMany(OrderProduct);
+OrderProduct.belongsTo(Product);
+
+module.exports = {Order, Product, User, OrderProduct};
+// OrderProduct.belongsTo(Order);

```

Comment:
`Through` tables are for `belongsToMany` this relationship is being defined wrong.